src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -25,12 +28,24 @@ class TimezoneType extends AbstractType implements ChoiceLoaderInterface
      * The choices are generated from the ICU function \DateTimeZone::listIdentifiers().
      *
      * @var ArrayChoiceList
+     *
+     * @deprecated since version 3.4, to be removed in 4.0
      */
     private $choiceList;
 
     /**
      * {@inheritdoc}
      */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        if ('datetimezone' === $options['input']) {
```

Comment:
I like this suggestion and we could also apply it for other `Date*Type` having this input option.
But I'd keep `datetimezone` as alias for consistency, as I'm not convinced deprecating old option values for other types would be worth it and such aliases are handful too (for configuration based form generation like in EasyAdmin for instance).